### ISSUE TITLE
About looping 'out.append()' until out.size == opt.sample_duration

### ISSUE THREAD

**Issue by iuhiyuh** [kenshohara/3D-ResNets-PyTorch#153]
I noticed that you were looping the samples with frames less than opt.sample_duration during training. If the sample with 'frames >= opt.sample_duration' in the test set. Will the impact on model performance be large?
